packages/nocodb/docker/litestream.yml

```diff
@@ -6,6 +6,7 @@ dbs:
       - type: s3
         endpoint: ${LITESTREAM_S3_ENDPOINT}
         force-path-style: true
+        region: ${LITESTREAM_S3_REGION}
```

Comment:
@pranavxc I think setting `region` does only make sense when ***not*** setting `endpoint` at the same time since `endpoint` already includes the region, cf. https://litestream.io/reference/config/#s3-replica.

Did you test what happens if `LITESTREAM_S3_REGION` is unset and `LITESTREAM_S3_ENDPOINT` is set and vice versa? Does Litestream ignore empty config entries?

Reply:
I tested and it was fine, will test again to verify

Reply:
Note that I hardcoded `force-path-style: true` in the Dockerfile.

About `force-path-style`:

> Uses the path style which is required by non-AWS services. This is automatically enabled if `endpoint` is set.

So I guess we'd have to remove `force-path-style: true` if we wanna support `region`, no?

(In the end I don't think it's really worth to support `region` since it only brings a tiny bit of convenience for AWS and Backblaze users but complicates support for other S3-compatible hosters.)

Reply:
Even with `force-path-style:` region was working for me.

Reply:
> Even with `force-path-style:` region was working for me.

Ok, interesting...

> I kept it because setting region is easier than figuring out the endpoint.

I see^^

I still think it would be best to adhere to what the official Litestream doc says, hence remove `force-path-style: true` if we keep `region` :)

Reply:
Ya better to remove since it automatically going to set to true when we are configuring custom endpoint